identity/app/views/profileForms.scala.html

```diff
@@ -51,7 +49,7 @@
     </li>
 }
 
-@content(i: Int, url: String, optionalClass: String = "", requiresValidation: Boolean = false)(body: => Html) = {
```

Comment:
not used

Reply:
I'm a big fan of comments by the committer!

Reply:
yes me too, I think it's a great habit to review your own PRs and comment. 🌟